src/assets/messages.json

```diff
@@ -1593,7 +1596,10 @@
     "contact-property-label": "Contact propriété",
     "contact-property-selector-placeholder": "Sélectionnez une propriété",
     "group-selector-placeholder": "Sélectionnez un groupe",
-    "fix-it": "Répare le",
+    "fix-issue": "Resoudre le probleme",
+    "locate-block-issue": "Repérez blocs à numéro fixe",
+    "flow-error-message": "Vous avez un problème de validation avec votre flux",
+    "block-error-message": "Vous avez bloc :block_count (s) qui a problème de validation.",
```

Comment:
@poojahpatil90 , when adding auto translated message in our localization spreadsheet with `dynamic` param like this, make sure to `copy-paste` the FR value to avoid a `space` before `:` (in this .json file, it's ok as you manually edited it, but google doc file still has `Vous avez: bloc block_count (s) qui a problème de validation.`

Reply:
Done